### ISSUE TITLE
Sort-amd-paths should consider "/" before anything else

### ISSUE THREAD

**Issue by obrejla** [cvisco/eslint-plugin-requirejs#70]
Because "/" is a dir separator.

This should NOT be warning:

``` js
define([
    'foo/bar/baz/Bat',
    'foo.bar/baz/Bat'
], function (bat1, bat2) {
    /* ... */
});
```

But currently is, but it makes sense to have a "shorter dirname" before longer, even though there is a "dot" in it.

**Comment by obrejla** [comment]
Hmm...sry, wrong example...

**Comment by obrejla** [comment]
Ok, example updated...missing callback params ;)

**Comment by obrejla** [comment]
It's not only about "dots". Slash should be "the smallest" i.e. "the most first" character, so e.g. this should NOT be warning as well:

``` js
define([
    'foo/bar/baz/Bat',
    'foo-bar/baz/Bat'
], function (bat1, bat2) {
    /* ... */
});
```

**Comment by obrejla** [comment]
I already work on a patch.

**Comment by obrejla** [comment]
Is it possible to make new minor release, please? As I said before, otherwise I can't use it at work and this fix is crucial for us ;) Thanks a lot!

**Comment by cvisco** [comment]
@obrejla done. I tried to do this last night but github was down for a while.

**Comment by obrejla** [comment]
Thanks a lot!
